src/pages/Subscriptions/SubscriptionsSettingsPage.vue

```diff
@@ -113,9 +120,35 @@ export default {
 	},
 	apollo: {
 		query: pageQuery,
-		preFetch(config, client) {
-			return client.query({
-				query: pageQuery
+		preFetch(config, client, { route, kvAuth0 }) {
+			return new Promise((resolve, reject) => {
+				client.query({
+					query: pageQuery
+				}).then(({ data }) => {
```

Comment:
This is copied over from the autolending settings page. As an aside, I really dislike this method of checking for authentication in components. Authentication checking should be handled by the router in navigation guards. I believe we are only doing this in 3 places at the moment, but I might tackle this as a refactor in the near future